pom.xml

```diff
@@ -271,7 +271,7 @@
 
                 <plugin>
                     <artifactId>maven-enforcer-plugin</artifactId>
-                    <version>1.4.1</version>
+                    <version>3.0.0-M1</version>
```

Comment:
Is this a temporary workaround until there is a GA release?

Reply:
Yes, explanation same as above